### PR TITLE
change TsFile config variable name

### PR DESCRIPTION
**Pull request by MyXOF**

### CODE REVIEW

File: src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java [hunk 1]
```diff
@@ -7,100 +7,117 @@
...
+    public double plaMaxError = 100;
     /**
-     * the default time series value is TS_2DIFF
+     * Default SDT max error is 100
      */
-    public String timeSeriesEncoder = "TS_2DIFF";
-    // public TSEncoding timeSeriesEncoder = TSEncoding.RLE;
+    public double sdtMaxError = 100;
+    
+    public double dftSatisfyRate = 0.1;
```
Comment (Beyyes) [inline src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java]: dftSatisfyRate has no comment

File: src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java [hunk 2]
```diff
@@ -7,100 +7,117 @@
  * @author kangrong
  */
 public class TSFileConfig {
+    // Memory configuration
     /**
-     * String encoder with UTF-8 encodes a character to at most 4 bytes.
+     * The memory size threshold for flushing to disk or HDFS, default value is
+     * 128 * 1024 * 1024
```
Comment (Beyyes) [inline src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java]: maybe 128 * 1024 * 1024 byte is better?

File: src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java [hunk 3]
```diff
@@ -7,100 +7,117 @@
...
-    public int pageCheckSizeThreshold = 100;
+    public int pageSize = 1024 * 1024;
     /**
-     * default data type of time is LONG
+     * The maximum number of data points in a page, defalut 1024*1024
      */
-    public String defaultTimeType = "INT64";
+    public int maxPointNumberInPage = 1024 * 1024;
+    
+    // Data type configuration
```
Comment (Beyyes) [inline src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java]: some beginning letter of comment is upper, but some is lower

Comment (MyXOF) [inline src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java]: Unified comment format

File: src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java (hunk 2 above)
Comment (MyXOF) [inline src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java]: now specified in the parameter name

File: src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java (hunk 1 above)
Comment (MyXOF) [inline src/main/java/cn/edu/thu/tsfile/common/conf/TSFileConfig.java]: add

**Pull request merged**